.github/workflows/build.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  pull_request:
+    branches: [ "main" ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Go ${{ matrix.go }} sample
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.15'
```

Comment:
go version used in go.mod and this is diff, is that ok ?

Reply:
Yeah that is okay. go.mod indicates that it can compile only in version >1.14

Reply:
ok so that is the min version